CONTRIBUTING.md

```diff
@@ -86,6 +86,9 @@ cd room
 # Run additional checks
 ./gradlew buildOnServer
 
+# Run coding standard & guideline checks
```

Comment:
@tikurahul @liutikas 
Do we know why `bOS` doesn't cover this?

Reply:
`ktlint` checks are performed using git hooks in AOSP. These hooks are installed automatically. We should try and do something similar.

Reply:
We can provide links to the commit hooks we use, but it's impossible to force someone to install them, so we should just add the check to our build imo

Reply:
Also is ktlint sufficient? Don't we also have error-prone, etc?